### ISSUE TITLE
Object Detection

### ISSUE THREAD

**Issue by BinYan-Wu** [amirgholami/adahessian#27]
Can the optimizer be used for Object Detection? I tested it myself and it seems that there will be an error
![image](https://user-images.githubusercontent.com/59214223/154906473-b7e35787-ce66-40f8-86da-e98d3697b8bf.png)

**Comment by yzlnew** [comment]
set `loss.backward(creat_graph=True)`.
